plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -53,26 +46,9 @@
 public abstract class BaseElasticsearchConnectorTest
         extends BaseConnectorTest
 {
-    protected final String image;
-    protected ElasticsearchServer elasticsearch;
-    protected RestHighLevelClient client;
+    protected abstract RestHighLevelClient getClient();
 
-    protected BaseElasticsearchConnectorTest(String image)
-    {
-        this.image = image;
-    }
-
-    @Override
-    protected QueryRunner createQueryRunner()
-            throws Exception
-    {
-        elasticsearch = new ElasticsearchServer(image, ImmutableMap.of());
-
-        HostAndPort address = elasticsearch.getAddress();
-        client = new RestHighLevelClient(RestClient.builder(new HttpHost(address.getHost(), address.getPort())));
-
-        return createElasticsearchQueryRunner(elasticsearch.getAddress(), TpchTable.getTables(), ImmutableMap.of(), ImmutableMap.of(), 3);
-    }
+    protected abstract void destroy();
```

Comment:
Why this is abstract method? This class does not call it.

Reply:
Removed it and got rid of the destroy method using `closeAfterClass` as you suggested.